joomla/joomla-cms
# [mod_login] Always use htmlspecialchars($str, ENT_COMPAT, 'UTF-8')

Pull Request for Issue #10399 .
#### Summary of Changes

Always use htmlspecialchars($str, ENT_COMPAT, 'UTF-8')
#### Testing Instructions
- Enable the mod_login module to the frontend
- see that it works (login)
- see that it works (logout)
- apply this patch
- see that it works (login)
- see that it works (logout)
